spec/features/login_visit_election_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+#feature 'Visit Election', js: true do
+feature 'Visit Election' do
+  let(:user) { create(:user) }
+  let(:election) { create(:election) }
+  let(:council) { create(:council) }
+  let(:post) { create(:post, council: council, elected_by: 'Terminsmötet') }
+	background do
+		council
+		post		
+		election
+		allow(election).to receive(:current_posts) { Post.all }
+	end
+
+	Steps 'Visit the election page' do
+		When 'visit the page' do
+			page.visit elections_path
+		end
+		Then 'I should see greeting' do
+    	page.should have_css('h1#election-title')
+    	find('h1#election-title').text.should include(election.title)
+		end
+	end
+  Steps 'Trying nominations page' do
+		When 'I try to see the page' do
+			page.visit new_nominations_path
+    end
+		Then 'I see alert' do
+      page.should have_css('div.alert-danger')
+      find('div.alert-danger').text.should include(I18n.t('unauthorized.new.nomination'))
+	  end
```

Comment:
`end` at 31, 3 is not aligned with `Then 'I see alert' do` at 28, 2<br>Tab detected.